### DISCUSSION TITLE
DAB+ not popular?

### DISCUSSION THREAD

**Discussion by skycamel** [rern/rAudio#1104] [Polls]
I find it puzzling that not more attention is given to the DAB+ feature.
This is a great and easy way to listen to digital radio with a minimum in investment.
All you need is a RTL-STR stick. For those with no or very low speed internet or with a usage limit DAB+ is the ideal solution.
Maybe in rural areas there's less choice in channels but in an urban area there are plenty.
I would like to know how come there's a lack of interest?

**Comment by R101G** [comment]
@skycamel 
rAudio evolved from a player which was aimed at "audiophiles", and it is probably still the case that most users want the best quality source they can get. Although DAB+ is a big improvement, most rAudio users will have an internet connection, and will choose the better quality and choice available from web streams. 

I have a DAB radio in the kitchen, which I think is where it belongs :)

**Reply by crazD802** [reply]
I hope that an RTL-STR stick will under the tree for me soon...

**Reply by skycamel** [reply]
@R101G you certainly have a point, but one doesn't exclude the other.
For people like myself, who listen to the radio the whole day (as background music) and have either a bad internet connection or limited data plan the DAB+ is a very interesting solution. With rAudio you can both use DAB+ or, if you like to have better quality, use internet radio or ex. flac files. How neat is that!
Myself I have two rpi, one with the Hifiberry Digi+ and one with the Hifiberry Amp3 which I use for background listening.
Unfortunately, my DAB+ is on the first rpi (because of the antenna connection) and I don't get snapcastclient to work on my second rpi so I can't listen to DAB+ as background on my Amp3 for the time being. I hope I get snapcastclient working.
Cheers,
Luc
